packages/scaffold-ui/src/partials/w3m-input-address/styles.ts

```diff
@@ -31,6 +31,7 @@ export default css`
     position: absolute;
     top: 50%;
     transform: translateY(-50%);
+    z-index: 2;
```

Comment:
The `<textarea>` had a higher z-index than the send input placeholder on iOS, causing clicks to be blocked. I've gone ahead and increased the z-index for input placeholder here 🙏